news/_posts/releases/2020-04-20-release-note-1765.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  "Release Of PrestaShop 1.7.6.5"
+subtitle: "Maintenance version of the 1.7.6.X branch"
+date:   2020-04-20 16:30:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/05/1765meta.jpg
+icon: icon-checked
+tags:
+ - 1.7
+ - 1.7.6.x
+ - version
+ - patch
+ - releases
+---
+
+
+As announced a few months ago, the PrestaShop core team has decided to make a forthright patch release process and deliver patch versions on a more regular basis, every time it’s needed. So here we are, 6 weeks after the release of 1.7.6.4, PrestaShop 1.7.6.5 is now available! 
```

Comment:
Maybe add a link to that article? I can't find it.

Reply:
was there an article about it ? i'm not sure

Reply:
the article about patch release process is actually in review in a pull request.

Reply:
```suggestion
A few months ago, the PrestaShop core team has decided to make a forthright patch release process and deliver patch versions on a more regular basis, every time it’s needed. So here we are, 6 weeks after the release of 1.7.6.4, PrestaShop 1.7.6.5 is now available! 
```